### ISSUE TITLE
screen output no longer seems to work in v0.14

### ISSUE THREAD

**Issue by dansteen** [bloonix/awesant#42]
Hey!   I just upgraded from v0.13 to v0.14, and the screen output no longer seems to work.  My config looks like this:
```
input {
    socket {
        type syslog
        host 127.0.0.1
        port 9998
        format json
        workers 10
        tags syslog
        add_field {
           env stag
           role api
        }
    }
}

output {
    redis {
        type syslog
        key syslog
        host <hostname>
        port 6379
    }
    screen {
        type syslog
        send_to stdout
    }
}

logger {
    #file {
    #    filename /var/log/awesant/agent.log
    #    maxlevel info
    #}
    screen {
        maxlevel info
    }
}
```
This worked fine in v0.13, and would print the log lines to the screen.

Thanks!

**Comment by j-schulz** [comment]
Fixed in 0.15.

**Comment by j-schulz** [comment]
Sorry that it took so long ... :)
